@ecomplus/storefront-components/src/js/TheCart.js

```diff
@@ -0,0 +1,99 @@
+import {
+  $ecomConfig,
+  i18n,
+  formatMoney
+} from '@ecomplus/utils'
+
+import ecomCart from '@ecomplus/shopping-cart'
+import APrices from './../APrices.vue'
+import CartItem from './../CartItem.vue'
+import DiscountApplier from './../DiscountApplier.vue'
+import ShippingCalculator from './../ShippingCalculator.vue'
+import RecommendedItems from './../RecommendedItems.vue'
+
+import {
+  i19checkout,
+  i19continueShopping,
+  i19discount,
+  i19emptyCart
+} from '@ecomplus/i18n'
+
+export default {
+  name: 'TheCart',
+
+  components: {
+    APrices,
+    CartItem,
+    DiscountApplier,
+    ShippingCalculator,
+    RecommendedItems
+  },
+
+  props: {
+    ecomCart: {
+      type: Object,
+      default () {
+        return ecomCart
+      }
+    },
+    lang: {
+      type: String,
+      default: $ecomConfig.get('lang')
+    },
+    checkoutUrl: {
+      type: String,
+      default: '/app/#/checkout'
+    },
+    amount: {
+      type: Object,
+      default () {
+        return {}
+      }
+    },
+    discountCoupon: String,
+    baseModulesRequestData: {
```

Comment:
Melhor manter o mesmo nome da [prop](https://github.com/ecomplus/storefront/blob/master/%40ecomplus/storefront-components/src/js/DiscountApplier.js#L60) do `DiscountApplier` (é o mesmo objeto) pra padronizar:

```suggestion
    modulesPayload: {
```